plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -127,6 +128,13 @@ func defaultPredicates() sets.String {
 				return predicates.NewMaxPDVolumeCountPredicate(predicates.GCEPDVolumeFilter, maxVols, args.PVInfo, args.PVCInfo)
 			},
 		),
+		// GeneralPredicates includes PodFitsResources, PodFitsHost, PodFitsHostPort and PodSelectorMatches
```

Comment:
I would replace this comment with "GeneralPredicates are the predicates that are enforced by all Kubernetes components (e.g. kubelet and all schedulers)"

The comment you currently have will get out of sync with the code over time because nobody will remember to update it when they add to predicates.GeneralPredictaes